Automattic/jetpack
# Add/my jetpack skeleton

This PR bootstraps the My Jetpack package, including a React app that will hold the whole My Jetpack page on admin.

For now, it's a very simple skeleton. Header and Footer are there as placeholders as we still need to work on these generic components and shared CSS styles.

#### Changes proposed in this Pull Request:

* Adds the skeleton of the My Jetpack package and JS app

#### Jetpack product discussion


p9dueE-3Ae-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make sure Jetpack and my-jetpack package are built
* Add `define( 'JETPACK_ENABLE_MY_JETPACK', true );` to your `wp-config.php`
* Make sure there's a `My Jetpack` submenu under the Jetpack top-level menu
* Look at the code organization and give feedback - it was mostly copied from the `connection-ui` package
